satellite/satellitedb/migrate.go

```diff
@@ -967,67 +299,47 @@ func (db *DB) PostgresMigration() *migrate.Migration {
-					`DROP TABLE project_payments CASCADE`,
-					`CREATE TABLE project_payments (
-						id bytea NOT NULL,
+
+					`CREATE TABLE project_invoice_stamps (
+						project_id bytea NOT NULL REFERENCES projects( id ) ON DELETE CASCADE,
+						invoice_id bytea NOT NULL UNIQUE,
+						start_date timestamp with time zone NOT NULL,
+						end_date timestamp with time zone NOT NULL,
+						created_at timestamp with time zone NOT NULL,
+						PRIMARY KEY ( project_id, start_date, end_date )
+					);`,
+
+					`CREATE TABLE project_members (
+						member_id bytea NOT NULL REFERENCES users( id ) ON DELETE CASCADE,
 						project_id bytea NOT NULL REFERENCES projects( id ) ON DELETE CASCADE,
-						payer_id bytea NOT NULL REFERENCES user_payments( user_id ) ON DELETE CASCADE,
-						payment_method_id bytea NOT NULL,
-						is_default boolean NOT NULL,
 						created_at timestamp with time zone NOT NULL,
+						PRIMARY KEY ( member_id, project_id )
+					);`,
+
+					`CREATE TABLE used_serials (
+						serial_number_id integer NOT NULL REFERENCES serial_numbers( id ) ON DELETE CASCADE,
+						storage_node_id bytea NOT NULL,
+						PRIMARY KEY ( serial_number_id, storage_node_id )
+					);`,
+
+					`CREATE TABLE user_credits (
+						id serial NOT NULL,
+						user_id bytea NOT NULL REFERENCES users( id ) ON DELETE CASCADE,
+						offer_id integer NOT NULL REFERENCES offers( id ),
+						referred_by bytea REFERENCES users( id ) ON DELETE SET NULL,
+						credits_earned_in_cents integer NOT NULL,
+						credits_used_in_cents integer NOT NULL,
+						expires_at timestamp with time zone NOT NULL,
+						created_at timestamp with time zone NOT NULL,
+						type text NOT NULL,
 						PRIMARY KEY ( id )
 					);`,
-				},
-			},
-			{
-				DB:          db.db,
-				Description: "Move InjuredSegment path from string to bytes",
-				Version:     41,
-				Action: migrate.SQL{
-					`ALTER TABLE injuredsegments RENAME COLUMN path TO path_old;`,
-					`ALTER TABLE injuredsegments ADD COLUMN path bytea;`,
-					`UPDATE injuredsegments SET path = decode(path_old, 'escape');`,
-					`ALTER TABLE injuredsegments ALTER COLUMN path SET NOT NULL;`,
-					`ALTER TABLE injuredsegments DROP COLUMN path_old;`,
-					`ALTER TABLE injuredsegments ADD CONSTRAINT injuredsegments_pk PRIMARY KEY (path);`,
-				},
-			},
-			{
-				DB:          db.db,
-				Description: "Remove num_redeemed column in offers table",
-				Version:     42,
-				Action: migrate.SQL{
-					`ALTER TABLE offers DROP num_redeemed;`,
-				},
-			},
-			{
-				DB:          db.db,
-				Description: "Set default offer for each offer type in offers table",
-				Version:     43,
-				Action: migrate.SQL{
-					`ALTER TABLE offers
-						ALTER COLUMN redeemable_cap DROP NOT NULL,
-						ALTER COLUMN invitee_credit_duration_days DROP NOT NULL,
-						ALTER COLUMN award_credit_duration_days DROP NOT NULL
-					`,
+					`CREATE UNIQUE INDEX credits_earned_user_id_offer_id ON user_credits (id, offer_id);`,
```

Comment:
this should be 
```
CREATE UNIQUE INDEX credits_earned_user_id_offer_id ON user_credits (id, offer_id)
WHERE credits_earned_in_cents=0;
```
But cockroachdb doesn't ssupport partial indexes atm, they have [this open issue](https://github.com/cockroachdb/cockroach/issues/9683)